### ISSUE TITLE
Undefined array key prevents login

### ISSUE THREAD

**Issue by BreadTeleporter** [phpvirtualbox/phpvirtualbox#337]
When logging in, I get a 500 Internal Server error, and when looking into apache2's logs I see:
```
[Tue Jan 16 11:01:10.592454 2024] [php:warn] [pid 10914] [client ::1:52246] PHP Warning:  Undefined array key "context" in /var/www/phpvirtualbox/endpoints/lib/language.php on line 77, referer: http://example.com
[Tue Jan 16 11:01:15.226285 2024] [php:error] [pid 10911] [client ::1:44216] PHP Fatal error:  Uncaught Error: Attempt to assign property "errors" on null in /var/www/phpvirtualbox/endpoints/api.php:338\nStack trace:\n#0 {main}\n  thrown in /var/www/phpvirtualbox/endpoints/api.php on line 338, referer: http://example.com
```
(domain name replaced for privacy reasons)

This is a fresh install and I'm (fairly) certain I have installed all of the required dependencies (php8.2-xml and php8.2-soap)
This is running on Debian 12, an Apache2 webserver, PHP 8.2, vboxwebsrv 7.0.12 and linux kernel version 6.1.0-15

**Comment by BreadTeleporter** [comment]
Fixed the first error by applying the patch in #292

**Comment by BreadTeleporter** [comment]
Fixed last error with #297
